AppCenter/AppCenter/MSACServiceAbstract.h

```diff
@@ -44,11 +45,9 @@
 - (void)updateConfigurationWithAppSecret:(NSString *)appSecret transmissionTargetToken:(NSString *)token;
 
 /**
- * Checks if the service needs the application secret.
- *
- * @return `YES` if the application secret is required, `NO` otherwise.
+ * The flag indicate whether the service needs the application secret or not.
  */
-- (BOOL)isAppSecretRequired;
+@property(atomic, readonly, getter=isAppSecretRequired) BOOL appSecretRequired NS_SWIFT_NAME(appSecretRequired);
```

Comment:
`isAppSecretRequired` is more "Swifty" than `appSecretRequired`.

https://swift.org/documentation/api-design-guidelines/#general-conventions

https://developer.apple.com/documentation/swift/objective-c_and_c_code_customization/renaming_objective-c_apis_for_swift#2990185

Reply:
@sindresorhus 👋 

Good timing. We're still discussing this internally